src/wayland/seat/keyboard/keymap_file.rs

```diff
@@ -12,12 +12,14 @@ use nix::{
 };
 use slog::error;
 
+#[allow(missing_docs)]
 #[derive(Debug)]
 pub struct KeymapFile {
     sealed: Option<SealedFile>,
     keymap: CString,
 }
 
+#[allow(missing_docs)]
```

Comment:
Are those allow missing docs lines still needed? Those types are now internal types not crate level public. If rustc still complains that those need docs then it's fine I guess.

Reply:
Apparently not :)